src/steps/02-EvaluationCriteria/MRR/OtherSupportingFactors.vue

```diff
@@ -16,9 +38,71 @@
 
 <script lang="ts">
 import Vue from "vue";
-import { Component } from "vue-property-decorator";
+import { Component, Mixins } from "vue-property-decorator";
+import { FairOpportunityDTO } from "@/api/models";
+import AcquisitionPackage from "@/store/acquisitionPackage";
+import { getYesNoRadioOptions, hasChanges } from "@/helpers";
+import _ from "lodash";
+import { RadioButton, YesNo } from "../../../../types/Global";
+import ATATRadioGroup from "@/components/ATATRadioGroup.vue";
+import ATATTextArea from "@/components/ATATTextArea.vue";
+import SaveOnLeave from "@/mixins/saveOnLeave";
 
-@Component({})
+@Component({
+  components: {
+    ATATRadioGroup,
+    ATATTextArea,
+  }
+})
 
-export default class OtherSupportingFactors extends Vue {}
-</script>
\ No newline at end of file
+export default class OtherSupportingFactors extends Mixins(SaveOnLeave) {
+  /* eslint-disable camelcase */
+  public exceptionDiscussion =""
+  public exceptionChoices :RadioButton[] = getYesNoRadioOptions("Exception")
+  public selectedException = ""
+
+
+  private get currentData(): FairOpportunityDTO {
+    return {
+      other_facts_to_support_logical_follow_on: this.selectedException,
+      other_facts_to_support_logical_follow_on_details: this.exceptionDiscussion,
+    } as FairOpportunityDTO;
+  }
+
+  private get savedData(): FairOpportunityDTO {
+    return {
+      other_facts_to_support_logical_follow_on: AcquisitionPackage.fairOpportunity?.
+        other_facts_to_support_logical_follow_on || "",
+      other_facts_to_support_logical_follow_on_details: AcquisitionPackage.fairOpportunity?.
+        other_facts_to_support_logical_follow_on_details || "",
+    } as FairOpportunityDTO;
+  }
+
+  private hasChanged(): boolean {
+    return hasChanges(this.currentData, this.savedData);
+  }
+
+  public async loadOnEnter(): Promise<void> {
+    const storeData = _.cloneDeep(AcquisitionPackage.fairOpportunity);
+    if (storeData) {
+      this.selectedException = storeData.other_facts_to_support_logical_follow_on||""
+      this.exceptionDiscussion = storeData.other_facts_to_support_logical_follow_on_details||""
+    }
+  }
+
+  protected async saveOnLeave(): Promise<boolean> {
```

Comment:
be sure to reset `this.exceptionDiscussion` to "" if user selects `NO` option in case had previously selected `YES` and entered/saved something